helpers.py

```diff
@@ -0,0 +1,10 @@
+def sanitize_input(input):
+    input_lower = input.lower()
+    input_list = input_lower.split()
+    for element in input_list:
+        element.strip()
+    input_sanitized = " ".join(input_list)
+    return input_sanitized
```

Comment:
Great idea to sanitize our inputs. 

Through some chaos testing I found that single quotes are breaking the queries. i.e. passing "Brendan's Workout"
I believe in SQL we can escape a single quote with two single quotes.

Reply:
Thank you, I can see the problem here. I added a if statement to handle that case.